aacgmv2/wrapper.py

```diff
@@ -151,7 +190,7 @@ def convert_latlon(in_lat, in_lon, height, dtime, code="G2A"):
         c_aacgmv2.set_datetime(dtime.year, dtime.month, dtime.day, dtime.hour,
                                dtime.minute, dtime.second)
     except:
```

Comment:
Yeah, I wasn't sure what errors it would raise and I didn't take the time to look into it.  I found ways to raise Type and Runtime errors with datetime, so added catches for those.

For the other exception, I added a log warning that contains the exception message raised that follows the method outlined on: https://wiki.python.org/moin/HandlingExceptions